content/updates/2017-08-02-release-notes.md

```diff
@@ -0,0 +1,39 @@
++++
+date = "2017-08-02"
+title = "Platform Release Notes: August 2, 2017"
++++
+
+Curious what’s new that you might find helpful as a cloud.gov user? Here are highlights from our platform updates over the past two weeks.
+<!--more-->
+
+Curious what’s new that you might find helpful as a cloud.gov user? Here are highlights from our platform updates over the past two weeks.
+
+### Added
+You can [add people to spaces](https://cloud.gov/docs/apps/managing-teammates/#give-roles-to-a-teammate) from the [dashboard](https://dashboard.fr.cloud.gov/#/) if you’re an Org Manager. For example, this makes it easier for program managers to add developers to projects without using the command line tool.
+
+### Changed
+In the dashboard, Org Managers and Space Managers can [remove a person from a space](https://cloud.gov/docs/apps/managing-teammates/#space-users) with one click on "Remove All Space Roles".
+Org managers must remove a person from all spaces before removing them from the Org otherwise. If you try to remove them with the dashboard, you’ll see an error if the user still belongs to any space.
+
+### In case you missed it
+We presented at two events during the Amazon Web Services Public Sector summit: [Deliver your agency mission faster with cloud.gov](https://www.youtube.com/watch?v=NGmDhWEfMuo&list=PLhr1KZpdzukePsKIUofhgp50b63-5yr1V&index=78) and [FedRAMP Accelerated: An update with GSA and cloud.gov](https://www.youtube.com/watch?v=iXqbIxtiwQY&index=87&list=PLhr1KZpdzukePsKIUofhgp50b63-5yr1V).
+
+### Platform releases
+We upgraded the Cloud Foundry deployment to [v268](https://github.com/cloudfoundry/cf-release/releases/tag/v268#v268-security-notices).
+
+You should [restage your application](http://cli.cloudfoundry.org/en-US/cf/restage.html) to [incorporate fixes in the base filesystem](https://docs.cloudfoundry.org/devguide/deploy-apps/stacks.html#cli-commands) and ensure you’re running the most recent language version supported by your [buildpack](https://docs.cloudfoundry.org/buildpacks/).
+
+#### Additional upgrades
+[RootFS cflinuxfs2 v1.141.0](https://github.com/cloudfoundry/cflinuxfs2/releases/tag/1.141.0) which remediates all 19 CVEs related to [USN-3363-1: ImageMagick vulnerabilities](https://usn.ubuntu.com/usn/USN-3363-1/)
```

Comment:
Missing bullet points for these items, so they consolidate together when rendered:

![screen shot 2017-08-02 at 12 27 11 pm](https://user-images.githubusercontent.com/391313/28891010-e2ae3a56-777d-11e7-8830-da6699598765.png)